docs/Gemfile.lock

```diff
@@ -0,0 +1,315 @@
+GEM
```

Comment:
We don't really need to check this one in. The Gemfile is sufficient to generate it. We can add it to .gitignore

Reply:
In general, it is considered goodness to check-in package lock files since it means that everyone will be using exactly the same versions of all the dependencies. It also means that you can control when you take updates to packages.